app/models/app.rb

```diff
@@ -58,6 +58,34 @@ def maintenance
     redis.get(maintenance_key)
   end
 
+  def whitelist
+    redis.smembers(whitelist_key) || []
+  end
+
+  def whitelist=(domains)
+    whitelist.each { |domain| remove_domain(domain) }
```

Comment:
Can we make it only add/remove the ones that actually changed? Otherwise there will be AuditEntry records created for every domain every time an update is made.